devito/ops/node_factory.py

```diff
@@ -0,0 +1,214 @@
+from sympy import Eq, Add, Mul, Pow, Integer, Float, Rational
+from devito import Dimension
+from devito.symbolics import Macro
+from devito.types import Symbol, Indexed, Array
+
+from devito.ops.utils import namespace
+
+
+class OpsNodeFactory():
+    """
+    Generates ops nodes for building an OPS expression.
+
+    Examples
+    --------
+    >>> a = OpsNodeFactory()
+    >>> b = a.new_symbol('symbol_name')
```

Comment:
instead of assigning it to ``b``, you should add another line displaying the returned value, as typically done in the Examples section.